dojo/engagement/views.py

```diff
@@ -269,7 +269,8 @@ def view_engagement(request, eid):
         Test.objects.filter(engagement=eng)
         .prefetch_related('tags', 'test_type')
         .annotate(count_findings_test_all=Count('finding__id'))
-        .annotate(count_findings_test_active_verified=Count('finding__id', filter=Q(finding__active=True)))
+        .annotate(count_findings_test_active=Count('finding__id', filter=Q(finding__active=True)))
```

Comment:
now this one is actually correct :-D

Reply:
It's not, it's actually an oversight. If so, then there is no longer a difference with verified at all.

Reply:
Typed too quickly, disregard.